pkg/azuredisk/controllerserver_v2.go

```diff
@@ -213,7 +215,12 @@ func (d *DriverV2) ControllerPublishVolume(ctx context.Context, req *csi.Control
 	}
 
 	caps := []*csi.VolumeCapability{volCap}
-	if !azureutils.IsValidVolumeCapabilities(caps) {
+	maxShares, err := azureutils.GetMaxShares(req.GetVolumeContext())
+	if err != nil {
+		return nil, status.Error(codes.InvalidArgument, "MaxShares value not supported")
```

Comment:
The error message makes it sound like `maxShares` is not support. A clear message would be: "invalid value specified by maxShares parameter"

BTW- Error messages should not start with capital letters per golint.

Reply:
Actually, it would be good to include the text of the error returned by `GetMaxShares`, e.g.

```go
return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("invalid value specified by maxShares parameter: %s", err))
```